src/Template/Template.WebAPI/Backend/Issues/Handlers/IssueCommandHandler.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Backend.Contracts.Issues.Commands;
+using Backend.Contracts.Issues.Events;
+using GoldenEye.Backend.Core.DDD.Commands;
+using GoldenEye.Backend.Core.DDD.Events;
+using GoldenEye.Backend.Core.Repositories;
+using GoldenEye.Shared.Core.Extensions.Mapping;
+
+namespace Backend.Issues.Handlers
+{
+    internal class IssueCommandHandler
+        : ICommandHandler<CreateIssue>
+    {
+        private IEventBus eventBus;
+
+        public IssueCommandHandler( IEventBus eventBus )
+        {
+            this.eventBus = eventBus;
+        }
+
+        public async Task Handle(CreateIssue message, CancellationToken cancellationToken)
```

Comment:
Could you change parameter name from `message` to `command`? It'd be imho more descriptive.